modules/test/playwright/tests/headless-builder-web/get.spec.ts

```diff
@@ -118,6 +120,14 @@ test('can see available path parameter properties of a singleElement endpoint',
 	await headlessBuilderPage.newApplicationTitleBox.fill('My-app');
 	await headlessBuilderPage.createApplicationButton.click();
 
+	const apiApplicationPage =
```

Comment:
In the line 98 of this file I still see:

```
await apiHelpers.objectEntry.deleteObjectEntryByExternalReferenceCode(
```

Is that still necessary?

Reply:
and in the line 252

Reply:
see https://github.com/liferay-headless/liferay-portal/pull/2322/commits/952ebba82152c69dc3f1ee4a6ecf90c0ca241ede